BUILD.gn

```diff
@@ -241,9 +241,19 @@ group("create_dist") {
         "build/mac:unsigned_dmg"
       ]
     } else {
-      deps += [
-        "build/mac:finalize_dmg_pkg"
-      ]
+      deps += [ "build/mac:finalize_dmg_pkg" ]
+
+      if (sparkle_eddsa_private_key != "") {
+        deps += [ "build/mac:generate_eddsa_sig_for_dmg" ]
+
+        if (build_delta_installer) {
+          deps += [ "build/mac:build_delta_installer" ]
```

Comment:
to extract delta signing into `mac:generate_eddsa_sig_for_delta` which calls `generate_eddsa_sig.py`

Reply:
done.